src/globals.css

```diff
@@ -268,10 +268,12 @@
   .shad-textarea {
     @apply h-36 bg-dark-3 rounded-xl border-none focus-visible:ring-1 focus-visible:ring-offset-1 ring-offset-light-3 !important;
   }
-
+  .bg-darkpink-500 {
+    background-color: #d02d5e; /* Replace with your desired dark pink color */
```

Comment:
Hey please use tailwind css 

just write bg-#d02d5e direct instead of background-color : #d02d5e